agent/lib/kontena/observable.rb

```diff
@@ -58,12 +61,14 @@ def add_observer(observer, observe)
     # Update @value to each Observer::Observe
     def notify_observers
       observers.each do |observe, observer|
-        begin
-          debug "notify: #{observer} <- #{@observable_value}"
+        if observer.alive?
+          debug "notify: #{observe} <- #{@observable_value.inspect[0..64] + '...'}"
 
           # XXX: is the Observable's Celluloid.current_actor guranteed to match the Actor[:node_info_worker] Celluloid::Proxy::Cell by identity?
           observer.async.update_observe(observe, Celluloid.current_actor, @observable_value)
-        rescue Celluloid::DeadActorError => error
```

Comment:
It seems like it was the `observer.to_s` that was raising the `Celluloid::DeadActorError`, and that `observer.async.*` can not raise `Celluloid::DeadActorError`